fetch_cluster_logs.bash

```diff
@@ -101,20 +116,25 @@ else
   done
 fi
 
+if [[ -z $max_artifact_size ]]; then
+  # default to 2GB
+  max_artifact_size=2000
+fi
+
 ssh_options="-A -T -l $ssh_user -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
 if [[ ! -z $identity_file ]]; then
   ssh_options="-i $identity_file ${ssh_options}"
 fi
 
 # download the DC/OS CLI
 if [ ! -f dcos-cli ]; then
-  wget https://downloads.dcos.io/binaries/cli/linux/x86-64/dcos-1.12/dcos --output-document=dcos-cli
+  wget https://downloads.dcos.io/cli/releases/binaries/dcos/linux/x86-64/0.7.5/dcos --output-document=dcos-cli
   chmod +x dcos-cli
 fi
 
 # link the CLI with the cluster
 if [[ $dcos_variant == "open" ]]; then
-  bash -c "./dcos-cli $debug_options cluster setup $master_public_ip --insecure $dcos_login_token_input"
+  bash -c "./dcos-cli $debug_options cluster setup $master_public_ip --provider=dcos-oidc-auth0 --insecure $dcos_login_token_input"
```

Comment:
nit: Same with `provider`.  Do not know if we'd ever want or need to change this, but it seems like it might be nice to have the option.